CleanSlate/common/scripted_score_values/04_succession_voting.txt

```diff
@@ -6884,4 +6892,4 @@ elector_candidate_opinion_eldership_vote = {
 			value >= 75
 		}
 	}
-}
\ No newline at end of file
+}
```

Comment:
Not sure the changes to this file are necessary. `has_law` should be valid within the title scope.

Reply:
It's valid in title scope, but since it's a realm law, it's only set on a ruler's primary title. Secondary titles may thus not have the same law as the primary title. Checking in the holder's scope checks the primary title of the holder, so that does indeed work properly.